app/javascript/availability/libraries_locations.json

```diff
@@ -324,7 +329,7 @@
     "MICROS-EG": "Engineering, 325 Hammond Bldg. - Microforms",
     "MICROS-EM": "Earth & Min. Sci., 105 Deike Bldg. - Microforms",
     "MICROS-HB": "Harrisburg - Microforms - 2nd Floor - In-Library Use",
-    "MICROS-HN": "Copy unavailable, request via Interlibrary Loan",
+    "MICROS-HN": "Hazleton - Microforms - Collection Temporarily Unavailable",
```

Comment:
same for this location, it is in the ill list and the label change here won't have an effect.

Reply:
now moved out of ILL list